go/consensus/tendermint/apps/staking/transactions.go

```diff
@@ -192,7 +192,7 @@ func (app *stakingApplication) addEscrow(ctx *api.Context, state *stakingState.M
 
 	// Check if sender provided at least a minimum amount of stake.
 	if escrow.Amount.Cmp(&params.MinDelegationAmount) < 0 {
-		return staking.ErrInvalidArgument
+		return staking.ErrUnderMinDelegationAmount
```

Comment:
Can we add a unit test that checks if this specific error is returned?

Reply:
Done! :)  I've also added a few extra tests for the other errors that can happen in `addEscrow`.